collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_fs.py

```diff
@@ -449,7 +445,9 @@ def create_fs(module, blade):
                             module.params["policy"], module.params["name"]
                         )
                     )
-    module.exit_json(changed=changed)
+    module.exit_json(
+        changed=changed, diff={"before": None, "after": convert_to_dict(fs_obj)}
```

Comment:
This doesn't seem to show the diff for a creation - I do see that this might be useful and would conform to Ansible descriptions.

Reply:
I'm not sure what you mean here. Have you tried this yourself and don't see a diff, or just think the code is wrong?

I ran this and pasted the output above, in my scenario it correctly printed a diff.

Reply:
I tried this in my lab with a new share creation and no diff was produced. The diff only appeared when doing an update to the share.

Reply:
Oo, interesting.

Could you provide the args you ran the module with? I'll recheck for me.